changelog.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+This file contains all the notable changes done to the Ballerina NATS package through the releases.
+
+## 1.1.0-alpha6 
```

Comment:
Shall we add the date here?

Reply:
Added 67f1e6cfc3a38cedc5fd9c30a302effb8c6ee327